internal/pkg/template/api/service/service.go

```diff
@@ -2,17 +2,55 @@ package service
 
 import (
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/template/api/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/template/api/gen/templates"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/repository/manager"
 )
 
 type Service struct {
-	dependencies dependencies.Container
+	dependencies        dependencies.Container
+	lock                *sync.Mutex
+	repositoriesManager *manager.Manager
```

Comment:
`manager.go` a `manager_test.go` by mohli ist priamo do `repository` pkg, tu by si mal potom `repository.Manager`, ... nie je tam asi ziaden dovod, mat to ako samostatny balicek (?)